gitbootcamp.rst

```diff
@@ -295,6 +295,15 @@ Creating a Pull Request
 
 6. Press the ``Create pull request`` button.
 
+You should include the issue number (``GH-NNNN``) in the title and in the
+body of the PR.
```

Comment:
It would be good to get Bedevere updated to automatically add it to the PR body:

https://github.com/python/bedevere/pull/430

Reply:
Bot can just link PR to issue. No need to copy `#NNNNN` to PR body to GitHub link them.

Reply:
The `GH-NNNN` in the PR body is needed to add a reference to the issue, whereas the `GH-NNNN` in the title is used by bedevere.

Note that the linked issues in the sidebar only list PRs that will close the issue (or the issues that will be closed by the PR).  There seems to be no mechanism to simply link PRs and issues other than mentioning the number and having a mention reference added by GitHub among the messages.

Reply:
> The default squash commit message is copied from the first commit in the PR.
PR title and body are not used in the default commit message at all.

Isn't this only the case for PRs that are submitted with only one commit, and without the new "Use PR title for commit message" option checked? At least, I _thought_ the squashed commit message was getting set correctly from the PR title on PRs with multiple commits, and it was just single-commit PRs that was the problem?

Reply:
Bedevere now adds the issue number -- I tweaked this sentence to only mention the title.